policy-library/samples/dnssec.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
```

Comment:
Should we be including all of these samples?

Reply:
It helps the user to have them at hand if they want to add some new constraint, but we cloud just add an explanation in the README with a link to the policy repo instead. 

What do you think would be better?

Reply:
That's a good point, although given that we are shipping a number of policies - are they really required, because the examples we include will serve as a basis to iterate upon?

Reply:
Makes sense. I will add instructions in the README on how to add constraints and remove the sample folder.